core/src/test/java/org/opensearch/sql/expression/aggregation/StdDevAggregatorTest.java

```diff
@@ -1,16 +1,9 @@
 /*
- *   Licensed under the Apache License, Version 2.0 (the "License").
- *   You may not use this file except in compliance with the License.
- *   A copy of the License is located at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- *   or in the "license" file accompanying this file. This file is distributed
- *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- *   express or implied. See the License for the specific language governing
- *   permissions and limitations under the License.
+ * Copyright OpenSearch Contributors
```

Comment:
The same.

Reply:
Checking why this has Apache 2 license.

Reply:
I guess both were by mistake: https://github.com/opensearch-project/sql/pull/142/commits/8632f80f349260b09273c250f9b82bb1a1afd02f